purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -788,9 +791,26 @@ class Purchases internal constructor(
     }
     // endregion
 
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    fun redeemWebPurchase(webRedemptionLink: DeepLink.WebRedemptionLink, listener: RedeemWebResultListener) {
+        purchasesOrchestrator.redeemWebPurchase(webRedemptionLink, listener)
+    }
+
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    sealed class DeepLink {
+        class WebRedemptionLink internal constructor(internal val redemptionToken: String) : DeepLink()
+    }
+
     // region Static
     companion object {
 
+        @ExperimentalPreviewRevenueCatPurchasesAPI
+        @JvmStatic
+        fun parseIntent(intent: Intent?): DeepLink? {
```

Comment:
Maybe this function could have some mention of "deep link" in the name so it's easy to find in IDE autocomplete? Some ideas: 
- `parseAsDeepLink()`
- `createDeepLink()`
- `buildDeepLink()`
- `getDeepLink()`

Reply:
Maybe we can also add an extension function that then calls this function? Something like 
```kotlin
fun Intent.parseAsDeepLink(): DeepLink?
```

There's [`@file:JvmMultifileClass`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-multifile-class/) but it doesn't seem to work great for our case.